### DISCUSSION TITLE
Family tree name

### DISCUSSION THREAD

**Discussion by iksdna** [PeWu/topola-viewer#162] [Ideas]
Hello,

first of all, congratulations on this great project.

Would it be possible to add an option to display the name of the family tree, which would display above the tree?
E.g. The ability to enter the name in the settings with a show/hide option. 
I think it would be a helpful option especially when exporting the tree to a file.
Or maybe there would be another option to add the tree name.

BR,
Andy

**Comment by FrankBuchholz** [comment]
Hi Andy,

Now you get at least the following (see commit https://github.com/PeWu/topola-viewer/commit/7c5203b2638b103b2ccb84b190feecb441a2ac21 ): The HEAD data including the name of the file from the GEDCOM file is displayed on the settings tab in the side panel. Ok, that's not exactly what you are asking for, as the name is not part of the graphics and therefore not part of any exported file. (I guess, that this would required a joint extension of topola, which renders the graph and topola-viewer, which shows the app.)

Greetings,
Frank
